raster/src/main/scala/geotrellis/raster/histogram/FastMapHistogram.scala

```diff
@@ -58,9 +60,9 @@ object FastMapHistogram {
   * The [[FastMapHistogram]] class.  Quickly compute histograms over
   * integer data.
   */
-class FastMapHistogram(_size: Int, _buckets: Array[Int], _used: Int, _total: Int)
+class FastMapHistogram(var size: Int, var buckets: Array[Int], var counts: Array[Long], var used: Int, var total: Long)
```

Comment:
That's important to fix, you have to set up your global email settings on your machine to your github account email so they link up.